Makefile

```diff
@@ -229,8 +229,8 @@ distclean::          ; rm -f build/Makefile-configuration
 #############################################################################
 # building coq:
 export PATH:=$(shell pwd)/sub/coq/bin:$(PATH)
-CONFIGURE_OPTIONS := -coqide "$(COQIDE_OPTION)" -with-doc no -local -no-custom
-BUILD_TARGETS := coqbinaries tools states ltac
+CONFIGURE_OPTIONS := -coqide "$(COQIDE_OPTION)" -with-doc no -prefix $(shell pwd)
+BUILD_TARGETS := coqbinaries tools states world
```

Comment:
@DanGrayson : Do you know how to get rid of the `world` target? It would be great to not compile all of Coq all the time.

Reply:
If we're supposed to build coq, as indicated by BUILD_COQ=yes, then I don't see a way to avoid the target "world".  I think it just means "all".

What part of coq do you think does not have to be built?

Reply:
I'll take a look ...